src/pages/coding-standards/docblock.md

```diff
@@ -128,7 +128,7 @@ Use the following templates for the license notice and copyright blocks:
 ```php
 <?php
 /**
- * Copyright © Magento, Inc. All rights reserved.
+ * Copyright &copy; Magento, Inc. All rights reserved.
```

Comment:
Is this correct? When I look at files in the magento codebase, I see the copyright character instead of the encoding.

Reply:
I don't remember changing this. It's probably an IDE auto-change. Thanks for catching.